### PR TITLE
test(Wallet_settings) Keycard test #88

### PR DESCRIPTION
**Pull request by Valentina1133**
Fixes https://github.com/status-im/desktop-qa-automation/issues/88 : 

- Moved test about Keycard from tst_wallet_accountsManagement to tst_wallet_settingsManagement
- Added class for KeycardView and 2 extra steps with checking Keycard View and its elements

### CODE REVIEW

File: test/ui-test/testSuites/global_shared/scripts/settings_names.py [hunk 1]
```diff
@@ -79,6 +79,17 @@ class SettingsSubsection(Enum):
...
+settingsContentBaseScrollView_StatusListItem = {"container": mainWindow_settingsContentBaseScrollView_StatusScrollView, "type": "StatusListItem", "unnamed": 1, "visible": True}
+settingsContentBaseScrollView_StatusListItem_2 = {"container": mainWindow_settingsContentBaseScrollView_StatusScrollView, "occurrence": 2, "type": "StatusListItem", "unnamed": 1, "visible": True}
+settingsContentBaseScrollView_StatusListItem_3 = {"container": mainWindow_settingsContentBaseScrollView_StatusScrollView, "occurrence": 3, "type": "StatusListItem", "unnamed": 1, "visible": True}
+settingsContentBaseScrollView_StatusListItem_4 = {"container": mainWindow_settingsContentBaseScrollView_StatusScrollView, "occurrence": 4, "type": "StatusListItem", "unnamed": 1, "visible": True}
+settingsContentBaseScrollView_StatusListItem_5 = {"container": mainWindow_settingsContentBaseScrollView_StatusScrollView, "occurrence": 5, "type": "StatusListItem", "unnamed": 1, "visible": True}
+settingsContentBaseScrollView_StatusListItem_6 = {"container": mainWindow_settingsContentBaseScrollView_StatusScrollView, "occurrence": 6, "type": "StatusListItem", "unnamed": 1, "visible": True}
```
Comment (anastasiyaig) [inline test/ui-test/testSuites/global_shared/scripts/settings_names.py:91]: any chance we can find these elements without relying on `occurrence `? Perhaps, adding object names for these options could be more stable solution

Comment (Valentina1133) [reply test/ui-test/testSuites/global_shared/scripts/settings_names.py]: Done

Comment (Valentina1133) [reply test/ui-test/testSuites/global_shared/scripts/settings_names.py]: Yes, I will definitely do that

**Comment by Valentina1133** [comment]
Closing this one because test was moved to a new framework

**Pull request closed without merging**